modules/net-vpc-swp/main.tf

```diff
@@ -33,7 +33,7 @@ resource "google_network_security_tls_inspection_policy" "tls-policy" {
   project               = var.project_id
   name                  = var.name
   location              = var.region
-  description           = var.tls_inspection_config.description
+  description           = var.tls_inspection_config.description != null ? var.tls_inspection_config.description : var.description
```

Comment:
you could do

`coalesce(var.tls_inspection_config.description , var.description)`

Reply:
Brilliant, thanks, added